.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -123,6 +123,10 @@ stages:
           inputs:
             artifactName: 'vhd-release-notes'
             targetPath: 'release-notes.txt'
+        - task: PublishPipelineArtifact@0
```

Comment:
nit: there are two stages with the same "PublishPipelineArtifact" name (the other for release notes). Perhaps the name can be changed?

Reply:
good point, will try a fix 👍